modules/spring-config/src/main/java/org/mule/runtime/config/spring/api/LazyComponentInitializer.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.config.spring.api;
```

Comment:
we can't have api with a package spring

Reply:
https://www.mulesoft.org/jira/browse/MULE-13592